ote/src/cljs/ote/ui/common.cljs

```diff
@@ -73,6 +73,11 @@
       (if component component [:span " "])
    ])
 
+(defn shortened-description [desc max-length]
+  (if (< max-length (count desc))
+    [:span (str (subs desc 0 max-length) "...")]
```

Comment:
Should we use unicode ellipsis \u2026 ?
https://www.fileformat.info/info/unicode/char/2026/index.htm